docs/cromwell_features/CallCaching.md

```diff
@@ -47,23 +41,82 @@ call-caching {
   invalidate-bad-cache-results = false
 
   blacklist-cache {
-    # The call caching blacklist cache is off by default. This is used to blacklist cache hit paths based on the
-    # prefixes of cache hit paths that Cromwell previously failed to copy for authorization reasons.
-    enabled: true
-    # Guava cache concurrency.
-    concurrency: 10000
-    # How long entries in the cache should live from the time of their last access.
-    ttl: 20 minutes
-    # Maximum number of entries in the cache.
-    size: 1000
+     # The call caching blacklist cache is off by default. This cache is used to blacklist cache hits based on cache
+     # hit ids or buckets of cache hit paths that Cromwell has previously failed to copy for permissions reasons.
+     enabled: true
+
+     # All blacklisting values below are optional except for groupings.workflow-option.
+
+     # A blacklist grouping can be specified in workflow options which will inform the blacklister which workflows
+     # should share a blacklist cache.
+     groupings {
+       workflow-option: call-cache-blacklist-group
+       concurrency: 10000
+       ttl: 2 hours
+       size: 1000
+     }
+
+     buckets {
+       # Guava cache concurrency.
+       concurrency: 10000
+       # How long entries in the cache should live from the time of their last access.
+       ttl: 20 minutes
+       # Maximum number of entries in the cache.
+       size: 1000
+     }
+
+     hits {
+       # Guava cache concurrency.
+       concurrency: 10000
+       # How long entries in the cache should live from the time of their last access.
+       ttl: 20 minutes
+       # Maximum number of entries in the cache.
+       size: 100000
+     }
```

Comment:
Seems like these doc examples should use the defaults used in `CallCachingBlacklistManager.scala`